ntbs-integration-tests/TransferPages/ActionTransferPageTests.cs

```diff
@@ -138,9 +142,12 @@ public async Task AcceptTransferAlert_CreatesTransferInAndOutTreatmentEventsForN
 
             var formData = new Dictionary<string, string>
             {
-                ["AcceptTransfer"] = "true",
-                ["TargetCaseManagerId"] = Utilities.CASEMANAGER_ABINGDON_ID.ToString(),
-                ["TargetHospitalId"] = Utilities.HOSPITAL_ABINGDON_COMMUNITY_HOSPITAL_ID
+                ["TransferRequest.AcceptTransfer"] = "true",
+                ["FormattedTransferDate.Day"] = "3",
+                ["FormattedTransferDate.Month"] = "12",
+                ["FormattedTransferDate.Year"] = "2021",
+                ["TransferRequest.TargetCaseManagerId"] = Utilities.CASEMANAGER_ABINGDON_ID.ToString(),
+                ["TransferRequest.TargetHospitalId"] = Utilities.HOSPITAL_ABINGDON_COMMUNITY_HOSPITAL_ID
             };

```

Comment:
Should we have tests for validation errors on this page too?
And maybe also a test where the transfer date changes? (I'm actually a bit unsure how the transfer date on the alert is being set currently... because I don't see it here I'm assuming the alert already exists with a null date here?)

Reply:
The test that is accepting the transfer is technically testing that the date is changing. The alert already exists with a null date yes, and that means the application will default to today's date on the page. 
Have added a test for date errors here too

Reply:
I think a test where the alert has a date on it and a different value is saved would still add value. You could imagine an implementation in which the date on the alert was taking precedence over the supplied date, for example.